CONTRIBUTING.md

```diff
@@ -37,31 +38,38 @@ This is a rough outline of how to prepare a contribution:
 - Make sure your commit messages are in the proper format (see below).
 - Push your changes to a topic branch in your fork of the repository.
 - If you changed code:
-   - add automated tests to cover your changes
+  - add automated tests to cover your changes
 - Submit a pull request to the original repository.
 
-## How to build and run the project
+### How to build and run the project
 
 Refer to the [building doc](site/building.md) to find out how to build from
 source.
 
-## How to run the test suite
+### How to run the test suite
 
 You can run the linting and unit tests by simply doing
 
 ```bash
 make test
 ```
 
-# Acceptance policy
+## Acceptance policy
 
 These things will make a PR more likely to be accepted:
 
- * a well-described requirement
- * tests for new code
- * tests for old code!
- * new code and tests follow the conventions in old code and tests
- * a good commit message (see below)
+- a well-described requirement
+- tests for new code
+- tests for old code!
+- new code and tests follow the conventions in old code and tests
+- a good commit message (see below)
+- All code must abide [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
+- Names should abide [What's in a name](https://talks.golang.org/2014/names.slide#1)
+- Code must build on both Linux and Darwin, via plain `go build`
+- Code should have appropriate test coverage, invoked via plain `go test`
```

Comment:
We're a bit more flexible than this suggests -- we have some integration tests that need external dependencies for example. But this'll do for now.

Reply:
`go test` on its own will just run tests in the current directory, and there are few tests in the top directory.

I think the advice is intended to mean "you should be able to run your tests using just go test with appropriate arguments", i.e., you don't need to use any other tooling. But it's ambiguous. Perhaps it would be clearer to say

> Tests should be written to work with `go test`

Do you think that resolves the ambiguity?

Reply:
Yes, sounds good.